mayastor-test/e2e/nightly/pvc_stress/pvc_stress_test.go

```diff
@@ -0,0 +1,366 @@
+// JIRA: CAS-500
+package pvc_stress_test
+
+import (
+	"context"
+	"fmt"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"strings"
+	"testing"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/deprecated/scheme"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"reflect"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var cfg *rest.Config
+var k8sClient client.Client
+var kubeInt kubernetes.Interface
+var k8sManager ctrl.Manager
+var testEnv *envtest.Environment
+var dynamicClient dynamic.Interface
+
+// Status part of the mayastor volume CRD
+type mayastorVolStatus struct {
+	state  string
+	reason string
+	node   string
+	/* Not required for now.
+	nexus struct {
+		children [ ]map[string]string
+		deviceUri string
+		state string
+	}
+	replicas []map[string]string
+	*/
+}
+
+func getMSV(uuid string) *mayastorVolStatus {
+	msvGVR := schema.GroupVersionResource{
+		Group:    "openebs.io",
+		Version:  "v1alpha1",
+		Resource: "mayastorvolumes",
+	}
+
+	msv, err := dynamicClient.Resource(msvGVR).Namespace("mayastor").Get(context.TODO(), uuid, metav1.GetOptions{})
+	if msv == nil {
+		return nil
+	}
+
```

Comment:
should check for err first?

Reply:
Agree, generally the 'proper' return values aren't valid when err != nil